documentation/modules/security/con-securing-client-authentication.adoc

```diff
@@ -6,8 +6,9 @@
 = User authentication
 
 [role="_abstract"]
-User authentication is configured using the `authentication` property in `KafkaUser.spec`.
-The authentication mechanism enabled for the user is specified using the `type` field.
+Use the `KafkaUser` custom resource to configure an authentication mechanism for clients that require access to a Kafka cluster. 
```

Comment:
I think this is a bit confusing. `e an authentication mechanism for clients` would be configured in the `Kafka` CR in the `.spec.kafka.listeners` section. This configures what kind of credentials will be issued for the user => TLS certificate, SCRAM-SHA password etc..

Reply:
Changed to:

> Use the `KafkaUser` custom resource to configure authentication credentials for users (clients) that require access to a Kafka cluster. 
> Configure the credentials using the `authentication` property in `KafkaUser.spec`.